roles/kubernetes/preinstall/tasks/0020-verify-settings.yml

```diff
@@ -44,6 +44,12 @@
     - { name: openstack_lbaas_enabled, value: "{{ openstack_lbaas_enabled }}" }
   ignore_errors: "{{ ignore_assert_errors }}"
 
+- name: Stop if kube-router chosen as network_plugin and kubeadm_enabled (not yet supported/verified)
```

Comment:
This is actually a problem. We are about to switch to use kubeadm as default deployment as you can see in #3301

Reply:
Cool, thanks for the note -- I've reworked logic, and verified ok
also with `kubeadm_enabled` installs (noting a _caveat_ re: kubeadm
lack of support for skipping kube-proxy deploy).